docs/source/08_integrations/1_prefect.md

```diff
@@ -0,0 +1,173 @@
+# Prefect
+
+Prefect is a workflow automation system. QHub integrates Prefect with a
+feature flag as follows:
+
+```yaml
+prefect: true
+```
+
+There are a bunch of components in getting Prefect working for you, here
+is a brief description of them:
+
+You also need a token/API Key to communicate to Prefect Cloud, same can be generate
```

Comment:
Would be useful to include some details on how to use the API key once you've generated it.

Reply:
added